File: app/src/main/java/com/my/televip/virtuals/messenger/NotificationCenter.java
package com.my.televip.virtuals.messenger;

import com.my.televip.Class.ClassNames;
import com.my.televip.Class.ClassLoad;
import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class NotificationCenter {


    private static int messagesDeleted = -1;
    private static int tlSchemeParseException = -1;


    public static int getMessagesDeleted() {
        if (messagesDeleted == -1) {
            messagesDeleted = XposedHelpers.getStaticIntField(ClassLoad.getClass(ClassNames.NOTIFICATION_CENTER), Obfuscate.getFieldName("NotificationCenter", "messagesDeleted"));
        }
        return messagesDeleted;
    }

    public static int getTlSchemeParseException() {
        if (tlSchemeParseException == -1) {
            tlSchemeParseException = XposedHelpers.getStaticIntField(ClassLoad.getClass(ClassNames.NOTIFICATION_CENTER), Obfuscate.getFieldName("NotificationCenter", "tlSchemeParseException"));
        }
        return tlSchemeParseException;
    }
}

File: app/src/main/java/com/my/televip/virtuals/messenger/NotificationCenter.kt
package com.my.televip.virtuals.messenger

import com.my.televip.clazz.ClassLoad
import com.my.televip.clazz.ClassNames
import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

object NotificationCenter {

    private var messagesDeleted = -1
    private var tlSchemeParseException = -1

    @JvmStatic
    fun getMessagesDeleted(): Int {
        if (messagesDeleted == -1) {
            val ncClass = ClassLoad.getClass(ClassNames.NOTIFICATION_CENTER) ?: return -1
            messagesDeleted = XposedHelpers.getStaticIntField(ncClass, Obfuscate.getFieldName("NotificationCenter", "messagesDeleted"))
        }
        return messagesDeleted
    }

    @JvmStatic
    fun getTlSchemeParseException(): Int {
        if (tlSchemeParseException == -1) {
            val ncClass = ClassLoad.getClass(ClassNames.NOTIFICATION_CENTER) ?: return -1
            tlSchemeParseException = XposedHelpers.getStaticIntField(ncClass, Obfuscate.getFieldName("NotificationCenter", "tlSchemeParseException"))
        }
        return tlSchemeParseException
    }
}
